### ISSUE TITLE
Failed to Fetch

### ISSUE THREAD

**Issue by ZhiyeTang** [microsoft/vscode#236677]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.96.0
- OS Version: Windows 23H2 and Ubuntu

Steps to Reproduce:

Ubuntu server failed to fetch vscode-server, and failed to fetch addons from vscode.

**Comment by ZhiyeTang** [comment]
Ubuntu server can not fetch the corresponding vscode-server either. Earlier today, my vscode on windows automatically upgraded to 1.96.2, then vscode posted that the server failed to fetch vscode-server. So I tried to roll back to this older version to ensure my connection.

**Comment by ZhiyeTang** [comment]
Thank you for all of your help. I've figured out this issue is caused by a vscode setting that should not be synced to my newly setup PC. Earlier this year, I set up a vscode proxy for my PC in mainland China, which enables the convenient downloading of vscode-server files. When I recently came to Hong Kong, this proxy setting synced to the newly-setup PC in Hong Kong, which led to the fetching failure of neither vscode-server files nor extensions. I would kindly suggest that VSCode provide more specific information regarding fetching failures, as it would greatly assist users in understanding and resolving issues.
